File: Assignments/Assignment_12/p5.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        for(int i=1;i<=row;i++){

                for(int j=0;j<row;j++){

                        if(j%2==0){

                                printf("%d\t",(i*i));
                        }else{

                                printf("%d\t",(i*i*i));
                        }
                }
                printf("\n\n");
        }
}

File: Flutter/Assignment/Square pattern set-2/p5.dart
import "dart:io";
void main(){
	print("Enter the no. of rows:");
	int? row=int.parse(stdin.readLineSync()!);
	for(int i=1;i<=row;i++){
		int x=i;
		for(int j=1;j<=i;j++){
			if(i%2!=0){
				stdout.write("${j}\t");
			}else{
				stdout.write("${x}\t");
				x--;
			}
		}
		print("");
	}
}
